packages/server/tests/acceptance/cacheService.spec.ts

```diff
@@ -22,74 +22,37 @@ import { expect } from 'chai';
 import { CacheService } from '../../../../packages/relay/src/lib/services/cacheService/cacheService';
 import { Registry } from 'prom-client';
 const registry = new Registry();
-const Redis = require('ioredis');
-
-const { hostname, port } = new URL(process.env.REDIS_URL || 'redis://127.0.0.1:6379');
-const redis = new Redis({
-  port: port,
-  host: hostname,
-});
 
 const DATA_LABEL_PREFIX = 'acceptance-test-';
 const DATA = {
   foo: 'bar',
 };
 const CALLING_METHOD = 'AcceptanceTest';
 
-const getConnectedClients = async () => {
-  const info = await redis.info('clients');
-  const infoRows = info.split('\n');
-  const connectedClientsRow = infoRows.find((row) => row.indexOf('connected_clients') !== -1);
-  if (connectedClientsRow) {
-    return parseInt(connectedClientsRow.split(':')[1]);
-  }
-
-  return 0;
-};
-
-const clearAllTestCacheFromRedis = async () => {
-  const keys = await redis.keys(`${DATA_LABEL_PREFIX}*`);
-  for (const key of keys) {
-    await redis.del(key);
-  }
-};
-
 describe('@cache-service Acceptance Tests for shared cache', function () {
   let cacheService: CacheService;
 
   before(async () => {
-    const pong = await redis.ping();
-    expect(pong).to.eq('PONG', 'redis server is running');
-
-    const connectedClientsBefore = await getConnectedClients();
     cacheService = new CacheService(global.logger, registry);
     await new Promise((r) => setTimeout(r, 1000));
-    const connectedClientsAfter = await getConnectedClients();
-    expect(connectedClientsAfter).to.eq(connectedClientsBefore + 1, 'successfully connected to redis server');
-
-    await clearAllTestCacheFromRedis();
-  });
-
-  afterEach(async () => {
-    await clearAllTestCacheFromRedis();
```

Comment:
@victor-yanev  Well, since I have changed the test and we're not using ioredis anymore, I don't see another way to get the redisCache instance through the CacheSevice and check if it is connected, since the redisCache property is private. If you have any suggestions I am open, maybe I can create some sort of method in the cacheService, but since it is not needed anywhere, I think it is redundant